inference-engine/tools/compile_tool/main.cpp

```diff
@@ -151,6 +148,10 @@ static std::map<std::string, std::string> parseConfig(const std::string& configN
 static std::map<std::string, std::string> configure(const std::string &configFile, const std::string &xmlFileName) {
     auto config = parseConfig(configFile);
 
+    IE_SUPPRESS_DEPRECATED_START
+        config[VPU_MYRIAD_CONFIG_KEY(PLATFORM)] = "VPU_MYRIAD_2480";
```

Comment:
@Maxim-Doronin is there a way to use option/value without macro and harding string value?

Reply:
btw, why deprecated option is used?

Reply:
To able to compile model without device in the system

Reply:
don't we have alternative option which is NOT deprecated?

Reply:
This option was used because its removal made it impossible to compile the model without a device.
As far as I know, there is no alternative yet.
We can solve it differently in the future, CF is too close for us to experiment (we can create new problems).
I think that for now this solution is not harmful, because we just brought back the old option.

Reply:
> As far as I know, there is no alternative yet.

@gladilov-gleb @Maxim-Doronin why?

Reply:
Deprecated option is used because it's the simplest and safest solution. I see it reasonable taking coming release into consideration.